tests/unittests/tests-senml_json/tests-senml_json.c

```diff
@@ -0,0 +1,562 @@
+/*
+ * Copyright (C) 2016 Lennart Dührsen <lennart.duehrsen@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include "embUnit/embUnit.h"
+
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <float.h>
+#include <math.h>
+
+#include "senml.h"
+
+#include "tests-senml_json.h"
+
+static char senml_json_empty_doc[]   = "[]";
```

Comment:
If https://github.com/RIOT-OS/RIOT/pull/5544/files#r97090882 is addressed these strings probably all can be `const`.